cdapython/factories/booleanquery/boolean_query.py

```diff
@@ -19,7 +19,7 @@ def _call_endpoint(
         limit: int,
         async_req: bool,
         include_total_count: bool,
-        show_term_count: Optional[bool],
+        show_counts: Optional[bool],
```

Comment:
two questions:
1) Is this needed on this factory or the other domain-type factories? (mutations, research_subject, subject, etc?). Do we use this?
2) Doc string at line 33 is incorrect and needs updating.

Reply:
In the factory, it is necessary to match the number of parameters in Q's 'Q._call_endpoint', which is currently being overrated by the factories. show in this error from pylint if show_counts is removed i can use a _ if we don't want to use show_counts `Number of parameters was 8 in 'Q._call_endpoint' and is now 7 in overriding 'BooleanQuery._call_endpoint`

Reply:
Right.  Thanks for reminding me about the Q._call_endpoint.

The docstring below at line 33 needs updating.